java-checks-test-sources/default/src/main/java/checks/RawExceptionCheckSample.java

```diff
@@ -24,7 +24,7 @@ public void throws_Error() {
 //            ^^^^^
   }
 
-  public void throws_Exception() throws Exception { // Noncompliant {{Define and throw a dedicated exception instead of using a generic one.}}
+  public void throws_Exception() throws Exception { // Noncompliant {{Replace generic exceptions with specific library exceptions or a custom exception.}}
```

Comment:
Why not all singular: `Replace this generic exception with specific library exception or a custom exception.`